pages/wiki.js

```diff
@@ -0,0 +1,144 @@
+import Head from 'next/head'
+import Navbar from '../components/Navbar';
+import Fuse from 'fuse.js';
+import links from '../links.json';
+import React, { useState } from 'react';
+
+function wiki() {
+
+  const [query, setQuery] = useState('');
+
+  const fuse = new Fuse(links, {
+    keys: [
+      'title',
+      'description',
+      'type'
+    ]
+  })
+
+  function searchInput({ currentTarget = {} }) {
+    const { value } = currentTarget;
+    setQuery(value);
+  }
+
+  const results = fuse.search(query);
+
+  const linkResult = query ? results.map(result => result.item) : links;
+
+  return (
+    <div className="container" >
+      <Head>
+        <title>Summer of Shipping</title>
+        <link rel="icon" href="/favicon.ico" />
+        <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet"></link>
+      </Head>
+
+      <main>
+        <img src="undraw_operating_system_4lr6.svg" className="SVG" />
+        <Navbar />
+        <div className="Searchbox">
+          <form className="search">
+            <input type="text" placeholder="Search Wiki.." value={query} onChange={searchInput} />
+          </form>
+        </div>
+
+        <div className="links">
+          {linkResult.map(card => {
+            const { title, description, type, link } = card;
+            return (
+              <a key={title} className="card" href={link}>
+                <h3>{title}</h3>
+                <p>{description}</p>
+              </a>
```

Comment:
If you think we might want to reuse these cards in the future, it might be worth pulling this out into its own component (not needed now, just optional)

Reply:
I think it would be better to integrate cards locally. The cards aren't anything but fancy div names, it's actually just 2 lines of CSS in the parent element. The rest of it is just custom styling for the cards. I think it'll be fine to have them generated through html/css